blog/2020-9-01-ensemble.md

```diff
@@ -0,0 +1,585 @@
+---
+id: object-detection-ensemble
+title: Ensembling multiple object detection models
+author: Savan Visalpara
+author_title: Machine Learning Engineer Intern
+author_url: https://github.com/savan77
+author_image_url: http://1.gravatar.com/avatar/0d1dd8e9d8d8c1198781c5fe8af36184
+tags: []
+---
+
+In the previous blog(<link>), we saw how we can add new models to train it directly from CVAT. Once the model has been trained you would also like to use it to run the inference. By default, CVAT supports inference on models supported by TensorFlow Object Detection API and Mask RCNN. Currently, the inference runs inside CVAT itself, which will be changed in the near future to serverless functions. So, it will be easier to add support for new models. But for now, you can create a Workflow that runs the inference and outputs an CVAT XML or any other file that you can upload to CVAT. You can also update the training Workflow to accept a flag for training/inference and do the same, if you prefer to have only one Workflow. In this blog post, we will see how you can create a Workflow to run inference on two different models running on different machines. To further demonstrate how you can add post-processing blocks, we will see how you combine output from multiple object detection models.
```

Comment:
Is there a previous blog to this? If so, link it like this `[previous blog](<link>)`

Reply:
yes, it will be done like that on Webflow. I didn't put actual links and images here since we have to move these to Webflow. I am adding those links here as well now.